bioimageio/spec/build_spec.py

```diff
@@ -74,24 +72,19 @@ def _get_weights(weight_uri, weight_type, source, root, **kwargs):
     if weight_type == "pytorch_state_dict":
```

Comment:
I am thinking can you read the weight format definition json file rather than hard code all these info? If necessary we can add more keys in to the weight format definition. What do you think?